pkgs/development/python-modules/azure-mgmt-media/default.nix

```diff
@@ -2,29 +2,27 @@
   lib,
   buildPythonPackage,
   fetchPypi,
+  setuptools,
   msrest,
-  msrestazure,
   azure-common,
   azure-mgmt-core,
-  pythonOlder,
 }:
 
 buildPythonPackage rec {
   pname = "azure-mgmt-media";
   version = "10.2.0";
-  format = "setuptools";
-
-  disabled = pythonOlder "3.6";
+  pyproject = true;
 
   src = fetchPypi {
     inherit pname version;
     extension = "zip";
     hash = "sha256-TVq/6dHttDGIUFzn8KTVeDTwcBMmphz3zrsGK7ux4aU=";
   };
 
-  propagatedBuildInputs = [
+  build-system = [ setuptools ];
+
+  dependencies = [
     msrest
-    msrestazure
     azure-common
     azure-mgmt-core
```

Comment:
`typing-extensions` is missing, see [typing-extensions](https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-media_10.2.0/sdk/media/azure-mgmt-media/setup.py#L76)

I think that for some `azure-*`modules `disabled = pythonOlder "3.9";` is set to avoid pulling in `typing-extensions`.

Reply:
Should I re-add the `pythonOlder` check then?

Reply:
Lots of them have that: https://github.com/search?q=repo%3ANixOS%2Fnixpkgs+disabled+%3D+pythonOlder+%223.9%22%3B+azure&type=code

I think it would be good to do so.